pkg/apis/core/v1/resource_tags.go

```diff
@@ -0,0 +1,59 @@
+package v1
+
+import (
+	"context"
+	"fmt"
+
+	"go.anx.io/go-anxcloud/pkg/api"
+	"go.anx.io/go-anxcloud/pkg/api/types"
+)
+
+func Tag(ctx context.Context, a api.API, obj types.IdentifiedObject, tags ...string) error {
```

Comment:
add doc comments for `Tag`, `Untag` and `ListTags` (or rather, all exported functions you ever add)